packages/application-shell/src/components/authenticated/__mocks__/am-i-logged-in.tsx

```diff
@@ -1,6 +1,6 @@
 type RenderFnArgs = { isAuthenticated: boolean };
 type Props = {
-  render: (args: RenderFnArgs) => JSX.Element;
+  render: (args: RenderFnArgs) => React.JSX.Element;
```

Comment:
Since we're already importing react in this file, I think it would be more convenient to import the `JSX` namespace instead of relying on the global React object.